src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,29 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/user';
+import userFromServer from '../../api/users';
+import './userInfo.css';
+
+type Props = {
+  userId: number,
+};
+
+export const UserInfo: React.FC<Props> = ({ userId }) => {
+  const user: User | undefined = userFromServer
+    .find(user1 => userId === user1.id);
```

Comment:
you shouldn't do it here, you should add `user` object to todo at the very start